benchmarks/01_trivial/01_aossoa.cpp

```diff
@@ -1,3 +1,9 @@
+/*
```

Comment:
NIT: Do we really so much vertical spacing? I would prefer to have it as compact as possible, i.e.:
```cpp
// Copyright (c) 2021-2024, Intel Corporation
// SPDX-License-Identifier: BSD-3-Clause
```

Reply:
I don't mind it. Changed to compact version.